src/plugins/inputs/join.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+var Chan = require("../../models/chan");
+var _ = require("lodash");
+
+exports.commands = ["join"];
+
+exports.input = function(network, chan, cmd, args) {
+	var client = this;
+	if (args.length !== 0) {
+		var irc = network.irc;
+		irc.join(args[0], args[1]);
```

Comment:
Is it reasonable to just forward the command as-is to `irc-framework`? Effectively doing what we do on master right now, but with `inputs/join.js` having some logic?

I understand the need for `inputs/join.js` is just because `irc-events/join.js` does not have access to the key.